### PR TITLE
Feature/StoryPromptsTable

### PR DESCRIPTION
**Pull request by ChadDiaz**
# Description

Fixes #

What work was done?

1. Created a StoriesNew table and migrated data from Stories "Title" row into StoriesNew.Title

Why was this work done?

1. It was a necessary step in order to restructure the database schema.

What feature / user story is it for?

1. Feature: Redesign Story Squad Schema || User Story: NA Backend Feature

Any relevant links or images / screenshots
N/A

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

- [x] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: data/migrations/20210802234758_storyPromptsTable.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('Story-Prompts', (sp) => {
```
Comment (lukecparr) [inline data/migrations/20210802234758_storyPromptsTable.js:2]: I don't think your table name follows the standard on this repo for using CamelCase.

**Review by lukecparr — CHANGES_REQUESTED** [review]
Please update the table name and the PR name to follow repo standards 🙂

File: data/migrations/20210802234758_storyPromptsTable.js (hunk 1 above)
Comment (ChadDiaz) [reply data/migrations/20210802234758_storyPromptsTable.js:2]: I think it looks consistent to most other compound table names.  Looks like 1 table is showing camel case but multiple others have the hyphenated naming case.

Comment (lukecparr) [reply data/migrations/20210802234758_storyPromptsTable.js:2]: Ah, yes. You're right about that. I got confused with what I was looking at. This is good then. 👍🏻

**Review by lukecparr — APPROVED** [review]

**Review by nico-herrera — APPROVED** [review]
everything checks out okay

**Pull request merged**